tests/testcore/onebox.go

```diff
@@ -427,6 +426,7 @@ func (c *TemporalImpl) startHistory() {
 				serviceName,
 				c.mockAdminClient,
 			),
+			fx.Provide(c.configProvider(primitives.HistoryService)),
```

Comment:
hah, I didn't realize history/matching/worker had no `*config.Config` in the test cluster fx graph